jdaviz/configs/cubeviz/plugins/parsers.py

```diff
@@ -11,7 +11,7 @@
 
 __all__ = ['parse_data']
 
-EXT_TYPES = dict(flux=['flux', 'sci'],
+EXT_TYPES = dict(flux=['flux', 'sci', 'data'],
                  uncert=['ivar', 'err', 'var', 'uncert'],
                  mask=['mask', 'dq'])
```

Comment:
Does this need updating too to accomodate `('QUALITY', 'mask-viewer')`?
```suggestion
                 mask=['mask', 'dq', 'quality'])
```

Reply:
Good call!